packages/axios-extension/src/abap/adt-catalog/handlers/transport.ts

```diff
@@ -55,10 +56,10 @@ function getTransportChecksResponse(doc: Document, xml: string, log: Logger): st
  *
  * @param doc
  * @returns
- * - For local package, an array list that contain a single empty string is returned.
+ * - For local package, return undefined.
  * - For errors or other unkonwn reasons no transport number found, an empty array list is returned.
  */
-function getTransportList(doc: Document): string[] {
+function getTransportList(doc: Document): TransportRequest[] | undefined {
```

Comment:
It would be easier for the consumer to have the errors thrown and an empty list for the case of local packages.

Reply:
Updated to return empty list for local package. Throw an error message for unexpected ADT response data format that cannot be processed.